lighthouse-core/gather/gatherers/dobetterweb/links.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const Gatherer = require('../gatherer');
+
+function getPageLinks() {
+  return new Promise((resolve, reject) => {
+    const linkList = [...document.querySelectorAll('a')]
```

Comment:
@ebidel I implemented your change and changed the file name to be more specific since this was residing in the `dobetterweb` folder. 

@paulirish I initially implemented this for the same reason but then the result returned by `evaluateAsync` kindda stripped down the `HTMLElement` object to an empty one. So I had to manually return the two attributes that I wanted to use. This kindda defeated the purpose of a common gatherer. So I did the changes as mentioned above.